go/parquet/file/column_reader_types.gen.go.tmpl

```diff
@@ -36,11 +36,13 @@ func (cr *{{.Name}}ColumnChunkReader) Skip(nvalues int64) (int64, error) {
       vals, _, err := cr.ReadBatch(batch,
         {{- if ne .Name "Boolean"}}
         {{.prefix}}.{{.Name}}Traits.CastFromBytes(buf),
+        arrow.Int16Traits.CastFromBytes(buf),
+        arrow.Int16Traits.CastFromBytes(buf))
         {{- else}}
         *(*[]bool)(unsafe.Pointer(&buf)),
+        nil,
+        nil)
         {{- end}}
-        arrow.Int16Traits.CastFromBytes(buf),
-        arrow.Int16Traits.CastFromBytes(buf))
```

Comment:
Wouldn't this use `nil` for *all* types rather than just for boolean? Does this bug exist for all types? or only for boolean?